source/manual/tag-content-with-facet-values.html.md

```diff
@@ -0,0 +1,33 @@
+---
+title: How to tag content with facet values
+parent: "/manual.html"
+layout: manual_layout
+section: Publishing
+type: learn
+owner_slack: "#govuk-platform-health"
+last_reviewed_on: 2019-03-2r9
+review_in: 3 months
+related_applications: [content-tagger]
+---
+
+Content tagger is the central place for tagging content with facet values.
+
+### How to edit facet values for a single page
```

Comment:
Not a blocker, but I think we should add a note that this should be performed by the content team.

Reply:
Done